digdag-cli/src/main/java/io/digdag/cli/Run.java

```diff
@@ -222,21 +222,9 @@ public SystemExitException usage(String error)
 
     private static final List<Long> USE_ALL = null;
 
-    public void run(String workflowNameArg, String matchPattern)
```

Comment:
This change is just extracting some part of `run` method to a method so that we can customize the bootstrap from test code.